src/client/components/forms/parts/publisher-data.js

```diff
@@ -76,14 +77,15 @@ class PublisherData extends React.Component {
 		let initialDisambiguation = null;
 		let initialAnnotation = null;
 		let initialIdentifiers = [];
+		console.log(this.props.publisher);
 
 		const prefillData = this.props.publisher;
 		if (prefillData) {
 			if (prefillData.area) {
 				initialArea = prefillData.area;
 			}
-			initialBeginDate = prefillData.beginDate;
-			initialEndDate = prefillData.endDate;
+			initialBeginDate = formatDate(new Date(prefillData.beginDate));
```

Comment:
Hmm can't do this here - these are partial dates, which can't be converted to dates (can be "YYYY") for example. They don't need formatting though, since they're already formatted strings.